tests/unit/utils/cassdc/containers.go

```diff
@@ -0,0 +1,32 @@
+package cassdc
+
```

Comment:
I like the reorganization! Just so you are aware a lot of these functions will likely be needed in reaper-operator and in medusa-operator. At some point soon we can pull the common test util code out into its own project.

Since I initially created these functions, I can add a follow up PR with some docs after yours is merged.